docs/content/direct/pr-signing.md

```diff
@@ -0,0 +1,119 @@
+     ```shell
+     $ gpg --list-secret-keys --keyid-format=long
+      /Users/hubot/.gnupg/secring.gpg
+      ------------------------------------
+      sec   4096R/3AA5C34371567BD2 2016-03-10 [expires: 2017-03-10]
+      uid                          Hubot <hubot@example.com>
+      ssb   4096R/4BB6D45482678BE3 2016-03-10
+
+     ```
+6. Copy your GPG key ID and run the command below, replacing ```[your_GPG_key_ID]``` with the key ID you just copied:
+   ```shell
+   gpg --armor --export [your_GPG_key_ID]
+   ```
+7. This should generate an output with your GPG key. Copy the characters starting from ```-----BEGIN PGP PUBLIC KEY BLOCK-----``` and ending at ```--END PGP PUBLIC KEY BLOCK-----``` (inclusive) to your clipboard.
+8. After copying or saving your GPG key, navigate to **Settings** in your Github
+9. Navigate to the **SSH and GPG keys** page under the Access section in the sidebar
+10. Under GPG keys, select **New GPG key**
+    - Enter a suitable name for your key under "Title" and paste your GPG key that you copied/saved in **Step 7** under "Key".
+    - Once done, click **Add GPG key**
+11. Your new GPG key should now be displayed under GPG keys.
+
+<br />
+
+## Setting up the SSH Key
+
+1. Open Git Bash and use the following command to generate your new SSH key (make sure to replace ```your_email``` with your Github-verified email address):
+   ```shell
+   ssh-keygen -t ed25519 -C "your_email"
+   ```
+   
+2. Press ```Enter``` to select the default option if prompted to set a save-file or passphrase for the key (you may choose to enter a passphrase if desired; this will prompt you to enter the passphrase everytime you perform a DCO sign-off).
+   - The following output should generate a randomart image 
+3. Use the following command to copy the new SSH key to your clipboard:
+   ```shell
+   clip < ~/.ssh/id_ed25519.pub
+   ```
+4. After copying or saving your SSH key, navigate to **Settings** in your Github.
+5. Navigate to the **SSH and GPG keys** page under the Access section in the sidebar.
+6. Under SSH keys, select **New SSH key**.
+   - Enter a suitable name for your key under "Title"
+   - Open the dropdown menu under "Key type" and select **Signing Key**
+   - Paste your SSH key that you copied/saved in **Step 3** under "Key"
+7. Your new SSH key should now be displayed under SSH keys.
+
+<br />
+
+## Creating Pull Requests
+
+Whether it's editing files from Kubestellar.io or directly from the Kubestellar Github, there are a couple steps to follow that streamlines the workflow of your PR:
+
+1. Changes made to any file are automatically committed to a new branch in your fork.
+   - When committing, make sure to specify the type of PR at the beginning of your commit message (i.e. :bug: if it addresses a bug-type issue)
+   - If the PR addresses a specific issue that has already been opened in the github, make sure to include the opened issue in **additional comments** (i.e. "fixes Issue #2XXX")
```

Comment:
"#2XXX" should be "#XXXX" where XXXX is the number of the issue the PR fixes. The reason for removing "2" is because soon we will be at 3000+ issues and etc.